src/app/services/user.service.ts

```diff
@@ -29,20 +32,20 @@ export class UserService {
   }
 
   logout(): any {
-    this.helper.removeStorageItem('email');
-    this.helper.removeStorageItem('firstName');
-    this.helper.removeStorageItem('lastName');
-    this.helper.removeStorageItem('address');
-    this.helper.removeStorageItem('creditCardNumber');
-    this.helper.removeStorageItem('cvv');
-    this.helper.removeStorageItem('role');
-    this.helper.removeStorageItem('userId');
+    localStorage.removeItem('email');
```

Comment:
any reason for this?

Reply:
is updated to be correct, just not showing here for some reason